src/shared/netif-naming-scheme.c

```diff
@@ -4,8 +4,9 @@
 #include "netif-naming-scheme.h"
 #include "proc-cmdline.h"
 #include "string-util.h"
+#include "strv.h"
 
-static const NamingScheme naming_schemes[] = {
+static NamingScheme naming_schemes[] = {
```

Comment:
Why drop `const`?

Reply:
As the parser update this global value... Ugh.